services/ui-src/src/components/drawers/ReportDrawer.tsx

```diff
@@ -1,55 +1,41 @@
-import { MouseEventHandler, useContext } from "react";
 // Components
 import { Box, Button, Flex } from "@chakra-ui/react";
 import { Spinner } from "@cmsgov/design-system";
-import { Drawer, Form, ReportContext } from "components";
+import { Drawer, Form } from "components";
 // utils
 import { useUser } from "utils";
 // types
-import { AnyObject, CustomHtmlElement, FormJson } from "types";
+import { AnyObject, CustomHtmlElement, FormJson, EntityShape } from "types";
 // constants
 import { closeText, saveAndCloseText } from "../../constants";
 
 export const ReportDrawer = ({
-  drawerDisclosure,
-  drawerTitle,
-  drawerInfo,
-  drawerDetails,
+  selectedEntity,
+  verbiage,
   form,
   onSubmit,
-  formData,
   submitting,
+  drawerDisclosure,
   ...props
 }: Props) => {
-  const { report } = useContext(ReportContext);
-
   // determine if fields should be disabled (based on admin roles)
   const { userIsAdmin, userIsApprover, userIsHelpDeskUser } =
     useUser().user ?? {};
   const isAdminTypeUser = userIsAdmin || userIsApprover || userIsHelpDeskUser;
   const buttonText = isAdminTypeUser ? closeText : saveAndCloseText;
 
   return (
-    <Drawer
-      drawerDisclosure={drawerDisclosure}
-      drawerTitle={drawerTitle}
-      drawerInfo={drawerInfo}
-      drawerDetails={drawerDetails}
-      {...props}
-    >
+    <Drawer verbiage={verbiage} drawerDisclosure={drawerDisclosure} {...props}>
       <Form
         id={form.id}
         formJson={form}
         onSubmit={onSubmit}
-        formData={formData ?? report?.fieldData}
+        formData={selectedEntity}
       />
       <Box sx={sx.footerBox}>
         <Flex sx={sx.buttonFlex}>
           {!isAdminTypeUser && (
-            <Button
-              variant="outline"
-              onClick={drawerDisclosure.onClose as MouseEventHandler}
-            >
+            <Button variant="outline" onClick={() => drawerDisclosure.onClose}>
```

Comment:
Looks like this breaks the Cancel button on my end:

https://user-images.githubusercontent.com/19439679/195607157-46c08b34-239a-4d5c-8db9-48dad30b7283.mov

Reply:
huh, weird. totally my bad. i'll switch it back. thank you!